nx-dev/data-access-documents/.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": ["@nrwl/web/babel"]
+  "presets": ["next/babel"],
```

Comment:
@FrozenPandaz Can you explain why you changed this?
As there's no migration for it (currently).